manuscripts/report.py

```diff
@@ -675,6 +676,22 @@ def period_name(cls, pdate, interval='quarter', offset=None):
 
         return name
 
+    def find_and_replace(self, directory, to_replace, replacement, file_type=None):
+        """
+        Looks for a string 'to_replace' in the 'directory' which has files containing
+        that string and replaces the string with its 'replacement' string. The type of
+        file to look for can be specified using 'file_type'.
```

Comment:
Add parameters in the comment. Have a look at how they are mentioned in Perceval, for example, since we should be using the same mechanism here.